error: #[construct_with] must have at least one argument
  --> tests/compile-fail/construct-with-errors.rs:20:7
   |
20 |     #[construct_with()] //~ ERROR #[construct_with] must have at least one argument
   |       ^^^^^^^^^^^^^^^^

error: unexpected literal in nested attribute, expected ident
  --> tests/compile-fail/construct-with-errors.rs:28:22
   |
28 |     #[construct_with("test")] //~ ERROR #[construct_with] should be of the form #[construct_with(<types>)]
   |                      ^^^^^^

error: #[construct_with] should be of the form #[construct_with(<primitive types>)]
  --> tests/compile-fail/construct-with-errors.rs:28:7
   |
28 |     #[construct_with("test")] //~ ERROR #[construct_with] should be of the form #[construct_with(<types>)]
   |       ^^^^^^^^^^^^^^^^^^^^^^
